bin/Update-AutoupdateManifests.ps1

```diff
@@ -1,75 +0,0 @@
-<#
```

Comment:
sorry, I'm aware this scripts doesn't fall under or fits to the SCOOP format and test but I still want it in my repo and under GIT...

Reply:
Not sure what the issue is here either :\ It's an auto formatter